Mathlib/Algebra/Group/Defs.lean

```diff
@@ -1160,6 +1160,50 @@ theorem CommGroup.toGroup_injective {G : Type u} : Function.Injective (@CommGrou
 #align comm_group.to_group_injective CommGroup.toGroup_injective
 #align add_comm_group.to_add_group_injective AddCommGroup.toAddGroup_injective
 
+/-- Define a Group structure on a Type by proving `∀ a, 1 * a = a` and
+`∀ a, a⁻¹ * a = 1`.
+See note [reducible non-instances]. -/
+@[to_additive (attr := reducible)]
+def Group.ofLeftAxioms {G : Type u} [Mul G] [Inv G] [One G]
```

Comment:
Can you add the `to_additive` docstring too?

I think it's probably worth a note like "note that this uses the default recursive implementation for `npow` and `zpow`" (and `nsmul`/`zsmul` for the additive version).